dlpy/applications/rcnn.py

```diff
@@ -119,6 +119,18 @@ def Faster_RCNN(conn, model_table='Faster_RCNN', n_channels=3, width=1000, heigh
         Specifies the architecture to be used as the feature extractor.
         Valid values: vgg16
         Default: vgg16, resnet50, resnet18, resnet34, mobilenetv1, mobilenetv2
+    random_flip : string, optional
+        Specifies how to flip the data in the input layer when image data is
+        used. Approximately half of the input data is subject to flipping.
+        Valid Values: 'h', 'hv', 'v', 'none'
+        Default: 'none'
```

Comment:
no default again

Reply:
the code signature indicates that random_flip=None, None is not a default value. It used to be 'none' that is a default value for that parameter but as it is now None (or NULL) - it is not default